ext/aws/module.lb

```diff
@@ -11,6 +11,35 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # -----------------------------------------------------------------------------
 
+class FreeRTOS_TCP(Module):
+    def init(self, module):
+        module.name = "tcp"
+        module.description = "FreeRTOS-Plus-TCP"
+
+    def prepare(self, module, options):
+        module.add_option(
+	        BooleanOption(
+	            name="dhcp",
+	            description="Use DHCP",
+	            default=False))
+        module.add_option(
+	        BooleanOption(
+	            name="dns",
+	            description="Enable simple DNS client/resolver",
+	            default=False))
```

Comment:
Are these options required in lbuild, or can be defined via `FreeRTOSIPConfigLocal.h`?

Reply:
I came up with the idea of the local config file after adding the options. Could be removed.